opentech/apply/funds/serializers.py

```diff
@@ -19,10 +20,23 @@ def to_representation(self, instance):
 
 class ReviewSummarySerializer(serializers.Field):
     def to_representation(self, instance):
+        reviews = instance.reviews.all()
         return {
-            'count': instance.reviews.count(),
-            'score': instance.reviews.score(),
-            'recommendation': instance.reviews.recommendation(),
+            'count': len(reviews),
+            'score': reviews.score(),
+            'recommendation': reviews.recommendation(),
+            'reviews': [
+                OrderedDict([
```

Comment:
a) dict is ordered by default now as of py3.6
b) Why do we need ordering?

Reply:
> a) dict is ordered by default now as of py3.6
It is a cpython implementation detail rather than the language feature as far as I know.

>  b) Why do we need ordering?
Not really, just seems like a practice of using ordered dictionaries in the APIs.